Resources/public/templates/tabs/details.hbt

```diff
@@ -0,0 +1,86 @@
+<h2>Content details</h2>
+<dl class="ez-details-authors ez-asynchronousview">
+    <dt>Creator</dt>
+    {{#if contentCreator }}
+
+        {{#if loadingError}}
+            <dd class="ez-asynchronousview-error ez-font-icon">
+                An error occurred while loading the creator.
+                <button class="ez-asynchronousview-retry ez-button ez-font-icon pure-button">
+                    Retry
+                </button>
+            </dd>
+        {{else}}
+            <dd>{{contentCreator.name}} - {{content.publishedDate}}</dd>
+        {{/if}}
+    {{else}}
+        {{#if loadingError}}
+            <dd class="ez-asynchronousview-error ez-font-icon">
+                An error occurred while loading the creator.
+                <button class="ez-asynchronousview-retry ez-button ez-font-icon pure-button">Retry</button>
+            </dd>
+        {{else}}
+            <dd class="ez-font-icon ez-asynchronousview-loading">Loading the creator</dd>
```

Comment:
you should not add the `ez-font-icon` and `ez-asynchronousview-loading` classes to the `dd` element, instead you should add a paragraph with such class, that way the `dd` element can be generated outside of the condition.